molecule/default/destroy.yml

```diff
@@ -63,8 +63,8 @@
     - name: Upload logs to object storage
       ignore_errors: True
       when:
-        - lookup('env', 'GITHUB_RUN_ID') is defined
-        - lookup('env', 'GITHUB_RUN_NUMBER') is defined
+        - lookup('env', 'GITHUB_RUN_ID')|length > 0
+        - lookup('env', 'GITHUB_RUN_NUMBER')|length > 0
```

Comment:
As a note for readers, this is a breaking change introduced in ansible-core 2.13. `lookup.env` has `default` as empty string which was before `undefined`